lms/djangoapps/courseware/tests/test_access_labster.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""
+Test labster for the access control framework
+"""
+from ccx_keys.locator import CCXLocator
+from student.models import CourseEnrollment
+from student.roles import CourseCcxCoachRole
+from courseware.tests.factories import UserFactory
+from xmodule.modulestore.tests.factories import CourseFactory
+from xmodule.modulestore.tests.django_utils import (
+    SharedModuleStoreTestCase,
+    TEST_DATA_SPLIT_MODULESTORE
+)
+import courseware.access as access
+from lms.djangoapps.ccx.models import CustomCourseForEdX
+
+# pylint: disable=protected-access
+
+class UserRoleTestCase(SharedModuleStoreTestCase):
+    """
+    Tests for user roles.
+    """
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    def make_coach(self, user, course_key):
+        """
+        Create coach user.
+        """
+        role = CourseCcxCoachRole(course_key)
+        role.add_users(user)
+
+    def make_ccx(self, coach, course, display_name):
```

Comment:
Cannot we simply inherit those 2 method from https://github.com/Livit/Livit.Learn.EdX/blob/labster/develop/openedx/core/djangoapps/labster/tests/base.py#L25 ?